neard/src/cli.rs

```diff
@@ -57,6 +58,10 @@ impl NeardCmd {
                 info!(target: "neard", "Removing all data and config from {}", home_dir.to_string_lossy());
                 fs::remove_dir_all(home_dir).expect("Removing data and config failed.");
             }
+            NeardSubCommand::StateViewer(cmd) => {
+                info!(target: "neard", "Invoking state-viewer");
```

Comment:
Do we need this log line? It doesn't add any new information to what the user typed themselves. The previous two lines print the path which is not neccessary passed in as a CLI argument, and they signify a dangerous information.

Meta: I feel that "overly chatty" tools are a big problem, as irrelevant printouts often make it hard to see the real error. To paraphrase Dijkstra, we should regard every line of output as a line spent.